codice/alliance
# CAL-94 replace Guest user with Videographer user

#### What does this PR do?
Child metacard (video chunks) are created with a Videographer user instead of the Guest user. This will allow administrators to fine tune permissions separate from the Guest user.
#### Who is reviewing it?
@stustison 
@rzwiefel 
@beyelerb 
@jrnorth 
@codice/security 

#### How should this be tested?
Build Alliance Experimental. Configure a streaming video endpoint. Ingest a streaming video. Search for the video chunks in the UI and verify that they uploaded as expected.
#### Any background context you want to provide?
The naming and package structure was heavily influenced by the naming in DDF. 
#### What are the relevant tickets?
CAL-94
#### Screenshots (if appropriate)
#### Checklist:
